Resources/Private/JavaScript/Host/Containers/ContentCanvas/InlineUI/MarkActiveNodeAsFocused/index.js

```diff
@@ -0,0 +1,32 @@
+import {Component} from 'react';
+import {connect} from 'react-redux';
+import {CR} from 'Host/Selectors/index';
+import {$transform} from 'plow-js';
+import style from './style.css';
+import {node as NodePropType} from 'Shared/PropTypes/CR/Node/index';
+
+@connect($transform({
+    focusedNode: CR.Nodes.focusedSelector
+}))
+export default class MarkActiveNodeAsFocused extends Component {
```

Comment:
It's perfectly fine to subscribe to store updates in redux, it's the core API. Wrapping some random js code into React component just for the sake of subscribing to Redux is not a very good practice I think.